packages/ui5-proxy-middleware/src/ui5/middleware.ts

```diff
@@ -46,13 +49,42 @@ function createRequestHandler(routes: { route: string; handler: RequestHandler }
     };
 }
 
-module.exports = async ({ options }: MiddlewareParameters<Ui5MiddlewareConfig>): Promise<RequestHandler> => {
+/**
+ * Search the project for the manifest.json.
+ *
+ * @param rootProject @ui5/fs reader collection with access to the project files
+ * @returns manifest.json as object or undefined if not found
+ */
+async function loadManifest(rootProject: ReaderCollection): Promise<Manifest | undefined> {
+    const files = await rootProject.byGlob('**/manifest.json');
+    if (files.length > 0) {
```

Comment:
From the documentaton of [`byGlob()`](https://sap.github.io/ui5-tooling/v3/api/@ui5_fs_ReaderCollection.html), it's not clear to me if it returns an empty array or undefined when there's no match. Also, do we need a `try/catch` block because of the `await` statement?

Reply:
It is wrapped in a try/catch in line 80.
I tried `byGlob` and it always returned an empty array if it didn't find anything, but to be sure, I added the undefined check with 53b4ed73